hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/accounts/AccountAccess.java

```diff
@@ -14,17 +14,19 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.spi;
+package com.hedera.node.app.spi.accounts;
 
+import com.hedera.node.app.spi.KeyOrLookupFailureReason;
 import com.hederahashgraph.api.proto.java.AccountID;
 import com.hederahashgraph.api.proto.java.ContractID;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Optional;
 
 /**
  * An interface used for looking up Keys on the account. NOTE: This class can be modified to return
  * any other fields needed from account object if needed in the future.
  */
-public interface AccountKeyLookup {
+public interface AccountAccess {
```

Comment:
I had added `getAccountById` to `AccountKeyLookup` and the name doesn't sound correct when it can provide account as well 🙂

Reply:
Open for any suggestions to any other names.

Reply:
> I had added getAccountById to AccountKeyLookup and the name doesn't sound correct when it can provide account as well
That's fair 🙂  Naming things is hard 😫